distributor-api-v1/operations.md

```diff
@@ -842,6 +843,7 @@ Gives a pricing information for the given configuration.
 |  | Property | Type | Description |
 | :--- | :--- | :--- | :--- |
 | `Client` | string | required | Identification of the client as described in [Authorization](https://mews-systems.gitbook.io/distributor-guide/distributor-api-v1/authorization). |
+| `ConfigurationId` | string | optional | GUID of the used Distributor configuration |
```

Comment:
@zdrazil 
- Add `.` at the end of sentence please.
- Also, we should probably add an info that it falls back to default configuration of the property if not provided.